src/Interactive/HostTest/InteractiveHostTest.csproj

```diff
@@ -101,8 +101,10 @@
     </ProjectReference>
   </ItemGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
This seems suspicious. Shouldn't the solution files just be updated to build this as AnyCPU instead of x86?